TCAT/Controllers/RouteOptionsViewController+Extensions.swift

```diff
@@ -208,9 +208,11 @@ extension RouteOptionsViewController: UITableViewDataSource {
 
         guard let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Cells.routeOptionsCellIdentifier, for: indexPath) as? RouteTableViewCell
             else { return UITableViewCell() }
-
-        cell.configure(for: routes[indexPath.section][indexPath.row], delegate: self)
-
+        
+        let routeId = routes[indexPath.section][indexPath.row].routeId
+        
+        cell.configure(for: routes[indexPath.section][indexPath.row], delayState: delayDictionary[routeId])
```

Comment:
nit, but can't we save `routes[indexPath.section][indexPath.row]` to a variable, rather than the `routeId`, and that'll be cleaner... so:

```
let a = routes[indexPath.section][indexPath.row]
cell.configure(for: a, delayState: delayDictionary[a.routeId])
```